test/parallel/test-net-connect-local-error.js

```diff
@@ -3,25 +3,40 @@ const common = require('../common');
 const assert = require('assert');
 const net = require('net');
 
+function getUnassignedPort(cb) {
+  const server = net.createServer();
+  server.listen({port: 0}, common.mustCall(() => {
+    // When the server is closed this port will no longer be assigned
+    const unassignedPort = server.address().port;
+    server.close(common.mustCall(() => {
```

Comment:
> IMHO port + 1 will always be flaky even in sequential.

I'm not sure I follow. Can you elaborate?
If you mean that there can be another test using `common.PORT + 1` (or `common.PORT` for that matter), I agree, but I think we'll be fine as long as there's no test on sequential listening/binding to `0` port.